docs/static/includes/interfaces/int.pe.input.bicep

```diff
@@ -1,17 +1,18 @@
 privateEndpoints: {
   {
-    roleAssignments : [...]
     lock: 'CanNotDelete'
-    tags: {...}
+    tags: {
+      'hidden-title': 'This is visible in the resource name'
+    }
     service: 'vault'
```

Comment:
Is this an example or a schema? if the latter, should we generalize the service value?

Reply:
It's the former, an example: 
![image](https://github.com/Azure/Azure-Verified-Modules/assets/5365358/bffe2e68-f154-4381-b243-b4e617177b8b)

Reply:
Thanks for elaborating, makes sense. Resolving